subprojects/gradle/gradle-logger/src/main/kotlin/com/avito/logger/destination/FileDestination.kt

```diff
@@ -0,0 +1,21 @@
+package com.avito.logger.destination
+
+import com.avito.logger.LogLevel
+import com.avito.logger.LoggingDestination
+import java.io.File
+
+internal class FileDestination(
```

Comment:
I didn't find where you create this destination. But it's strange that you don't use a metada here

Reply:
it is not used anymore; but i hesitate to remove it right now
removed now